test/env/setup_env.sh

```diff
@@ -106,6 +106,9 @@ case "$SUNDIALS_TEST_TYPE" in
         ;;
 esac
 
+# Disable GTest in CI until we determine SEGFAULT cause in test_sundials_errors.c
+export SUNDIALS_TEST_DISABLE_GTEST=ON
```

Comment:
I prefer the DISABLE name in this case because (to me) it implies that you should not turn off GTest unless you have a reason to.

Reply:
I don't think we document any of the `SUNDIALS_TEST` options. Was that on purpose, or should I add them all?

Reply:
Since these are not documented currently (I assume we chose to do this because we did not want to bother users with developer-specific options), that we leave it be in this PR, and add it to our list of "developer documentation enhancements" that we want to make.

Reply:
Ah, you're right we don't document those currently and I don't see a reason to add them to the install guide since users shouldn't need to set these. So this can wait for a later PR.